framework/src/org/checkerframework/framework/type/treeannotator/ListTreeAnnotator.java

```diff
@@ -28,7 +31,15 @@
      */
     public ListTreeAnnotator(TreeAnnotator... annotators) {
         super(null);
-        this.annotators = Collections.unmodifiableList(Arrays.asList(annotators));
+        List<TreeAnnotator> annotatorList = new ArrayList<>();
+        for (TreeAnnotator annotator : annotators) {
```

Comment:
Is this flattening of the sub-ListTreeAnnotators just for performance? Why is it necessary now?